storage/rocksdb/rdb_i_s.cc

```diff
@@ -864,7 +869,7 @@ static int rdb_i_s_compact_stats_fill_table(
       field[0]->store(cf_name.c_str(), cf_name.size(), system_charset_info);
       field[1]->store(level_str.c_str(), level_str.size(), system_charset_info);
       field[2]->store(type_str.c_str(), type_str.size(), system_charset_info);
-      field[3]->store(value, true);
+      field[3]->store(std::stod(value));
```

Comment:
Did you intend to remove the true indicating the value is unsigned?

Reply:
Yes, the overridden method with boolean accept longlong but the value we pass is actually double. Essentially this diff fixes a bug here. Please see a Test Plan note in "Check that compaction level metrics are still published to
INFORMATION_SCHEMA.ROCKSDB_COMPACTION_STATS properly including real double values" section where previously the Score was displayed as 0 and now it is 0.25.